apps/design/backend/src/app.test.ts

```diff
@@ -201,7 +204,82 @@ test('Update layout options', async () => {
   });
 });
 
-test('Export election package', async () => {
+test('Election package management', async () => {
+  const baseElectionDefinition =
+    electionFamousNames2021Fixtures.electionDefinition;
+  const { apiClient, workspace } = setupApp();
+
+  const electionId = (
+    await apiClient.createElection({
+      electionData: baseElectionDefinition.electionData,
+    })
+  ).unsafeUnwrap();
+
+  const electionPackageBeforeExport = await apiClient.getElectionPackage({
+    electionId,
+  });
+  expect(electionPackageBeforeExport).toEqual({});
+
+  // Initiate an export
+  await apiClient.exportElectionPackage({ electionId });
+  const electionPackageAfterInitiatingExport =
+    await apiClient.getElectionPackage({ electionId });
+  expect(electionPackageAfterInitiatingExport).toEqual({
+    task: {
+      createdAt: expect.any(Date),
+      id: expect.any(String),
+      payload: `{"electionId":"${electionId}"}`,
+      taskName: 'generate_election_package',
+    },
+  });
+  const taskId = assertDefined(electionPackageAfterInitiatingExport.task).id;
+
+  // Check that initiating an export before a prior has completed doesn't trigger a new background
+  // task
+  await apiClient.exportElectionPackage({ electionId });
+  const electionPackageAfterInitiatingRedundantExport =
+    await apiClient.getElectionPackage({ electionId });
+  expect(electionPackageAfterInitiatingRedundantExport).toEqual(
+    electionPackageAfterInitiatingExport
+  );
+
+  // Complete an export
+  await processNextBackgroundTaskIfAny({ log: jest.fn(), workspace });
```

Comment:
As mentioned in the last commit, calling this allows us to incorporate background task processing into app tests without having to fully start up a background worker